Proposals/state-of-the-community.md

```diff
@@ -0,0 +1,36 @@
+# State of The Community
+
+* Proposal: [PR#n]([link to PR])
+* Authors: [Matt Dias](@mdiasdev)
+* Review Manager: [Name]([github profile])
+* Status: [Proposed]()
+
+## Introduction
+
+There has been an interesting shift in the community. Where people used to gather around iOS or macOS as topics, everyone seems to have instead moved to all gather around Swift. I think it would be interesting to hear from a group of community builders and members about the state of the community, what it means to them, and how others can help keep up the spirit of building together.
+
+## Motivation
+
+To discuss the state of the community, how it has changed, how it should change, and help others get involved.
+
+## Proposed contribution
+
+I can represent a community leader (I help organize a Meetup). Otherwise, I'm happy to help reach out to potential hosts.
+
+## Source compatibility
+
+This podcast is for the community by the community. As such, I think it's important to regularly take a health check and reevaluate the state of things to seek out areas of improvement -- a retro of sorts.
+
+## Alternatives considered
+
+I think the existence of the podcast is a nod to the state of the community (in that is thriving). However, I don't think I've heard representatives from the community discuss the community itself in an attempt to motivate others to get involved.
+
+## Roles
+TBD
+
+- Lead Host:
+- Co-hosts:
```

Comment:
Have you found any co-hosts? I can do it of course but I think it would be better if we found other voices. Do you have anyone in mind? Have you reached out to anyone? Who should we try to get?

Reply:
Hi! I would like to contribute and though I don't have radio material voice, this conversation is very interesting to chime in. For context: I run the Swift Miami Meetup with my friend @chuva-io. I also happen to work with @mdiasdev.

Reply:
After thinking about it more, I think it'd be cool to have:
- Community Leader 
  - likely me unless there's someone better (totally up for anyone)
- Content Creator 
  - I don't know any to reach out to personally
- Community Member
  - I just asked the other Learn Swift City leaders if they have anyone in mind
  - if not, I have a couple Bostonians (though I'd like to spread out the view)

It would also be pretty interesting to hear Chris' opinion about the communities from a language creator perspective

Reply:
I think this is great @mdiasdev . Can you update the proposal to include these details? Then let's try to fill in the specific roles.